src/viewers/viewer/Viewer.js

```diff
@@ -446,37 +446,31 @@ class Viewer extends OlObject {
         });
 
         // we might have some requested defaults
-        var initialTime =
-        this.getInitialRequestParam(REQUEST_PARAMS.TIME);
-        initialTime =
-        initialTime !== null ? (parseInt(initialTime)-1) :
+        var initialTime = this.getInitialRequestParam(REQUEST_PARAMS.TIME);
+        initialTime = initialTime !== null ? (parseInt(initialTime)-1) :
         this.image_info_['rdefs']['defaultT'];
         if (initialTime < 0) initialTime = 0;
         if (initialTime >= dims.t) initialTime =  dims.t-1;
-        var initialPlane =
-        this.getInitialRequestParam(REQUEST_PARAMS.PLANE);
-        initialPlane =
-        initialPlane !== null ? (parseInt(initialPlane)-1) :
+        var initialPlane = this.getInitialRequestParam(REQUEST_PARAMS.PLANE);
+        initialPlane = initialPlane !== null ? (parseInt(initialPlane)-1) :
             this.image_info_['rdefs']['defaultZ'];
         if (initialPlane < 0) initialPlane = 0;
         if (initialPlane >= dims.z) initialPlane =  dims.z-1;
-        var initialCenterX =
-        this.getInitialRequestParam(REQUEST_PARAMS.CENTER_X);
-        var initialCenterY =
-        this.getInitialRequestParam(REQUEST_PARAMS.CENTER_Y);
+        var initialCenterX = this.getInitialRequestParam(REQUEST_PARAMS.CENTER_X);
+        var initialCenterY = this.getInitialRequestParam(REQUEST_PARAMS.CENTER_Y);
+        let imgCenter;
         if (initialCenterX && !isNaN(parseFloat(initialCenterX)) &&
-        initialCenterY && !isNaN(parseFloat(initialCenterY))) {
-        initialCenterX = parseFloat(initialCenterX);
-        initialCenterY = parseFloat(initialCenterY);
-        if (initialCenterY > 0) initialCenterY = -initialCenterY;
-        if (initialCenterX >=0 && initialCenterX <= dims['width'] &&
-            -initialCenterY >=0 && -initialCenterX <= dims['height'])
-        imgCenter = [initialCenterX, initialCenterY];
+                    initialCenterY && !isNaN(parseFloat(initialCenterY))) {
+            initialCenterX = parseFloat(initialCenterX);
+            initialCenterY = parseFloat(initialCenterY);
+            if (initialCenterY > 0) initialCenterY = -initialCenterY;
+            if (initialCenterX >=0 && initialCenterX <= dims['width'] &&
+                    -initialCenterY >=0 && -initialCenterX <= dims['height']) {
```

Comment:
This should be `-initialCenterY <= dims['height']`

Reply:
Fixed in 3928450